eo-maven-plugin/src/main/java/org/eolang/maven/OyRemote.java

```diff
@@ -81,4 +96,55 @@ public Input get(final String name) throws MalformedURLException {
         return new InputOf(url);
     }
 
+    /**
+     * File downloader.
+     * @param link Link to file
+     * @param name Location
+     * @throws IOException If fails
+     */
+    private static void download(final String link, final String name)
+        throws IOException {
+        final URL url = new URL(link);
+        try (InputStream in = url.openStream()) {
+            Files.copy(in, Paths.get(name), StandardCopyOption.REPLACE_EXISTING);
+        }
+    }
+
+    /**
+     * Hash of head master.
+     * @return SHA of commit
+     * @throws IOException if fails
+     * @checkstyle NonStaticMethodCheck (20 lines)
+     */
+    private String getSha() throws IOException {
+        String sha = "master";
+        final String link = String.format(
+            "%s%s",
```

Comment:
@yasamprom let's use this URL instead: `https://home.objectionary.com/tags.txt` (should work already)